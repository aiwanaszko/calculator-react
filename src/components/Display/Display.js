import React from "react";
import EquationRow from "../EquationRow/EquationRow";
import ResultRow from "../ResultRow/ResultRow";

export default function Display(props) {
  return (
    <div>
      <EquationRow />
      <ResultRow />
    </div>
  );
}
