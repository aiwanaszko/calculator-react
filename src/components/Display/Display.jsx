import React from "react";
import EquationRow from "../EquationRow/EquationRow";
import ResultRow from "../ResultRow/ResultRow";

export default function Display(props) {
  return (
    <div className="display">
      <EquationRow value={props.equation} />
      <ResultRow value={props.result} />
    </div>
  );
}
