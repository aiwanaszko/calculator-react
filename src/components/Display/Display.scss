.display {
  border-radius: 5px;
  margin: 0 auto;
}
