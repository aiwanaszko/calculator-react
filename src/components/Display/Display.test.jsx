import React from "react";
import renderer from "react-test-renderer";
import Display from "./Display";

it("renders the Display correctly", () => {
  const component = renderer.create(<Display />).toJSON();
  expect(component).toMatchSnapshot();
});
