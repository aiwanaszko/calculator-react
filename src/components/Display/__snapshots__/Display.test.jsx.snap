// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Display correctly 1`] = `
<div
  className="display"
>
  <div
    className="equation-row"
  >
    <input
      readOnly={true}
      type="text"
    />
  </div>
  <div
    className="result-row"
  >
    <input
      readOnly={true}
      type="text"
    />
  </div>
</div>
`;
